AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+|--|--|--|--|
+|JobId|int|Job ID for the scan (YYYYMMDD).|20230414|
+|RoleName|string|The role definition name.|reader|
+|Type|string|The role definition type.|CustomRole|
+|Description|string|A basic description of the role and its permissions.|Lets you view everything, but not make any changes|
+|AssignableScopes|string|The role definition assignable scopes.|["/"]|
+|Permissions|string|The role definition permissions.|[{"actions": ["*/read"], "notActions": [], "dataActions": [], "notDataActions": []}]|
+|CreatedOn|datetime|Date and time of creation.|14/04/2023 23:38:05|
+|UpdatedOn|datetime|Date and time of the last update.|14/04/2023 23:38:05|
+|CreatedBy|string|ID of the user who created the role definition.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|UpdatedBy|string|ID of the user who updated the role definition.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|Id|string|The role definition ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/providers/Microsoft.Authorization/roleDefinitions/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|RoleDefinitionName|string|The role definition name.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+
+___
+
+### **Secure score control**
+
+|Field Name|Field Type|Description|Sample value|
+|--|--|--|--|
+|OrgTenantId|string|The tenant ID.|00x000xx-00x0-00xx-00xx-0x0xx000xx00|
+|subscriptionid|string|The subscription ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|Id|string|The resource Id.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/providers/Microsoft.Security/secureScores/ascScore/secureScoreControls/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|Name|string|The resource name.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|DisplayName|string|User friendly display name of the control.|Restrict unauthorized network access|
+|Weight|int|Relative weight for this specific control in each of your subscriptions. Used when calculating an aggregated score for this control across all of your subscriptions.|0|
+|Type|string|The control type.|microsoft.security/securescores/securescorecontrols|
+|CurrentScore|double|The current score.|0.0|
+|PercentageScore|double|Ratio of the current score divided by the maximum. Rounded to 4 digits after the decimal point.|0.0|
+|MaxScore|string|The maximum score available.|0|
+|NotApplicableResourceCount|string|Number of not applicable resources in the control.|3|
+|UnhealthyResourceCount|int|Number of unhealthy resources in the control.|0|
+|HealthyResourceCount|int|Number of healthy resources in the control.|0|
+|SourceType|string|The type of security control.|BuiltIn|
+|AssessmentDefinitions|string|Array of assessments metadata IDs that are included in this security control.|["/providers/Microsoft.Security/assessmentMetadata/00000xxx-xx00-0x00-x0x0-00xx00000x00", "/providers/Microsoft.Security/assessmentMetadata/00000xxx-xx00-0x00-x0x0-00xx00000x00"]|
+|JobId|int|Job ID for the scan (YYYYMMDD).|20230414|
+|TotalResourceCount|int|The total number of resources in the control.|3|
+
+___
+
+### **Secure score**
```

Comment:
Secure Scores

![image](https://user-images.githubusercontent.com/84017721/233036485-e044ea30-1bcd-4fad-96c4-280bd918f2f9.png)

Reply:
Updated all the table names according to this